docs/debugger/create-custom-views-of-native-objects.md

```diff
@@ -88,6 +88,30 @@ The Visual Studio debugger loads *.natvis* files in C++ projects automatically,
 >[!NOTE]
 >Natvis rules loaded from a *.pdb* apply only to the types in the modules that the *.pdb* refers to. For example, if *Module1.pdb* has a Natvis entry for a type named `Test`, it only applies to the `Test` class in *Module1.dll*. If another module also defines a class named `Test`, the *Module1.pdb* Natvis entry does not apply to it.
 
+**To install and register a *.natvis* file via a VSIX package:**
+
+A VSIX package can install and register *.natvis* files. No matter where are they installed, all registered *.natvis* files are automatically picked up during debugging.
+
+1. Include the *.natvis* file in the VSIX package. For example in a project file:
```

Comment:
```suggestion
1. Include the *.natvis* file in the VSIX package. For example, for the following project file:
```

Reply:
This change feels weird. I just want to give an example to show that the natvis file needs to be included in a VSIX. There are other ways to do so.

Reply:
Actually never mind. Feels fine after read it again.